examples/chip-tool/commands/clusters/ReportCommand.h

```diff
@@ -528,10 +528,11 @@ class SubscribeEvent : public ReportCommand
         ReportCommand("subscribe-event", credsIssuerConfig),
         mClusterIds(1, clusterId), mEventIds(1, eventId)
     {
-        AddArgument("attr-name", eventName);
-        AddArgument("min-interval", 0, UINT16_MAX, &mMinInterval);
-        AddArgument("max-interval", 0, UINT16_MAX, &mMaxInterval);
-        AddArgument("keepSubscriptions", 0, 1, &mKeepSubscriptions);
+        AddArgument("attr-name", eventName, 0, "Event name.");
```

Comment:
Why is the argument named `attr-name` if it's an event ? Suggest renaming it `event-name`